packages/picasso.js/src/core/chart/index.js

```diff
@@ -677,6 +677,46 @@ function chartFn(definition, context) {
     setInteractions();
   };
 
+  /**
+   * Layout the chart with new settings and / or data
+   * @param {object} [def] - New chart definition
+   * @param {Array<DataSource>|DataSource} [def.data] Chart data
+   * @param {ChartSettings} [def.settings] Chart settings
+   * @param {string[]} [def.excludeFromUpdate=[]] Keys of components to not include in the update
+   * @experimental
+   */
+  instance.layout = (newProps = {}) => {
```

Comment:
How about calling the function `layoutComponents` to make it a bit more clear what it actually does, and to avoid confusion with "layout" that we often use when we talk about the data model...

Reply:
Good point! Updated!